projects/components/src/load-async/wrapper/load-async-wrapper.component.ts

```diff
@@ -66,12 +66,12 @@ export class LoadAsyncWrapperComponent {
         this.loaderType = this.config?.load?.loaderType;
         break;
       case LoadAsyncStateType.NoData:
-        this.icon = this.config?.noData?.icon ?? IconType.NoData;
+        this.icon = this.config?.noData?.showIcon ?? true ? this.config?.noData?.icon ?? IconType.NoData : undefined;
```

Comment:
🤨

Reply:
hah... want me to simplify it?

Reply:
we are kind of assigning a default value here in the same line.

Reply:
I think this would read clearer if you didn't use the first nullish coalescing operator and just compared to `undefined` instead.

Reply:
addressed